tests/static_lib_arch_flags_configs/app_static_lib_srcs_flags_configs/src/app.c

```diff
@@ -1,5 +1,6 @@
 #include "static0.h"
-#include "static1.h"
+#include "static0_archive.h"
+#include "static1_archive.h"
```

Comment:
This looks incorrect. The app shouldn't be able to include this.

static0_archive() should be in lib_static0/api/static0.h

Reply:
Reverted.